sqlite/ext/comdb2/tablepermissions.c

```diff
@@ -98,6 +98,8 @@ static int permissionsOpen(sqlite3_vtab *p, sqlite3_vtab_cursor **ppCursor){
   bdb_user_get_all(&pCur->ppUsers, &pCur->nUsers);
   unlock_schema_lk();
   *ppCursor = &pCur->base;
+  if (pCur->nUsers == 0)
+      return SQLITE_EMPTY;
```

Comment:
This isn’t right. IMHO, an empty table is perfectly normal, and shouldn’t be treated as error.

Reply:
Well, this table can only have content when user schema is turned on. This was meant to be like an indicator for that. But I agree that this looks not too good.